<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositView</class>
 <widget class="QDialog" name="DepositView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(252, 248, 232);</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>360</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(236, 179, 144);
border-radius: 20px;</string>
   </property>
   <widget class="QLabel" name="label_deposit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сумма вклада</string>
    </property>
   </widget>
   <widget class="QLabel" name="term_placement">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Срок размещения</string>
    </property>
   </widget>
   <widget class="QLabel" name="beginning_term">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Начало срока</string>
    </property>
   </widget>
   <widget class="QLabel" name="Interest_rate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Процентная ставка</string>
    </property>
   </widget>
   <widget class="QLabel" name="frequency_payments">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>209</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Периодичность выплат</string>
    </property>
   </widget>
   <widget class="QLabel" name="capital_interest">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>241</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Капитализация
процентов</string>
    </property>
   </widget>
   <widget class="QLabel" name="deposits">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>278</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Пополнения</string>
    </property>
   </widget>
   <widget class="QLabel" name="partial_withdrawals">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>318</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Частичные снятия</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_term_placement">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
background-color: rgb(252, 248, 232);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_frequency_payments_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>318</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
	color: black;
  	font-size:14px;
  	border:none;
	border-radius: 5px;
	background-color: rgb(252, 248, 232);
}

QComboBox::hover {
	background-color: rgb(212, 226, 212);
	border-radius: 5px;
}

QComboBox::drop-down {
      subcontrol-origin: padding;
      subcontrol-position: top right;
      width: 15px;
      border:none;
 }

QComboBox QAbstractItemView {
	background:rgba(255,255,255,1);
    border:1px solid rgba(228,228,228,1);
    border-radius:0px 0px 5px 5px;
	 font-size:14px;
    outline: 0px;
}

QComboBox QAbstractItemView::item{
	height:36px;
	color:red;
	padding-left:9px;
	background-color:red;
}</string>
    </property>
    <item>
     <property name="text">
      <string>Разовое</string>
     </property>
    </item>
   </widget>
   <widget class="QDateEdit" name="dateEdit_refill">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>278</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(252, 248, 232);</string>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>1</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2022</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_add_remove">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>318</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(252, 248, 232);</string>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>1</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2022</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add_refill">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>278</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(212, 226, 212);
    border-radius: 5px;
    border-color: beige;
}
QPushButton:pressed {
    background-color: rgb(182, 205, 182);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add_remove">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>318</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(212, 226, 212);
    border-radius: 5px;
    border-color: beige;
}
QPushButton:pressed {
    background-color: rgb(182, 205, 182);
    border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_date_form">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
	color: black;
  	font-size:14px;
  	border:none;
	border-radius: 5px;
	background-color: rgb(252, 248, 232);
}

QComboBox::hover {
	background-color: rgb(212, 226, 212);
	border-radius: 5px;
}

QComboBox::drop-down {
      border:0px;
 }

QComboBox::drop-arow {
	color: black;
}


QComboBox QListView {
	 background:rgba(255,255,255,1);
    border:1px solid rgba(228,228,228,1);
    border-radius:0px 0px 5px 5px;
	 font-size:14px;
    outline: 0px;
}
</string>
    </property>
    <item>
     <property name="text">
      <string>месяцев</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>лет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>дней</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit_input_tax">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
background-color: rgb(252, 248, 232);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="term_placement_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Налоговая ставка</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEdit_label_deposit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_capital_interest">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>253</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(236, 179, 144);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Interest_rate">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	color: black;
  	font-size:14px;
  	border:none;
	border-radius: 5px;
	background-color: rgb(252, 248, 232);
}

QComboBox::hover {
	background-color: rgb(212, 226, 212);
	border-radius: 5px;
}

QComboBox::drop-down {
      subcontrol-origin: padding;
      subcontrol-position: top right;
      width: 15px;
      border:none;
 }

QComboBox QAbstractItemView {
	background:rgba(255,255,255,1);
    border:1px solid rgba(228,228,228,1);
    border-radius:0px 0px 5px 5px;
	 font-size:14px;
    outline: 0px;
}

QComboBox QAbstractItemView::item{
	height:36px;
	color:red;
	padding-left:9px;
	background-color:red;
}</string>
   </property>
   <item>
    <property name="text">
     <string>Фиксированная</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Зависит от суммы</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Зависит от срока</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_frequency_payments">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>221</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	color: black;
  	font-size:14px;
  	border:none;
	border-radius: 5px;
	background-color: rgb(252, 248, 232);
}

QComboBox::hover {
	background-color: rgb(212, 226, 212);
	border-radius: 5px;
}

QComboBox::drop-down {
      subcontrol-origin: padding;
      subcontrol-position: top right;
      width: 15px;
      border:none;
 }

QComboBox QAbstractItemView {
	background:rgba(255,255,255,1);
    border:1px solid rgba(228,228,228,1);
    border-radius:0px 0px 5px 5px;
	 font-size:14px;
    outline: 0px;
}

QComboBox QAbstractItemView::item{
	height:36px;
	color:red;
	padding-left:9px;
	background-color:red;
}</string>
   </property>
   <item>
    <property name="text">
     <string>Раз в месяц</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в день</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_percent">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>140</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>140</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;</string>
   </property>
   <property name="text">
    <string> % годовых</string>
   </property>
  </widget>
  <widget class="QLabel" name="rubil">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>  руб.</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>1</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2022</year>
     <month>11</month>
     <day>28</day>
    </datetime>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::MonthSection</enum>
   </property>
   <property name="calendarPopup">
    <bool>false</bool>
   </property>
   <property name="currentSectionIndex">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_deposits">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>288</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_partial_withdrawals">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>328</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_go">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>376</y>
     <width>125</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(212, 226, 212);
    border-radius: 20px;
    border-color: beige;
}
QPushButton:pressed {
    background-color: rgb(182, 205, 182);
    border-style: inset;
}</string>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>429</y>
     <width>391</width>
     <height>162</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(236, 179, 144);
border-radius: 20px;</string>
   </property>
   <widget class="QLabel" name="label_accrued_interest">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>13</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Начисленные проценты</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_accrued_interest">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>13</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
background-color: rgb(252, 248, 232);
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_amount_end">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>108</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Сумма на вкладе к концу
срока</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_amount_end">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>113</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
background-color: rgb(252, 248, 232);
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tax">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>63</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Налог</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_tax">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>63</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
background-color: rgb(252, 248, 232);
</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>570</width>
     <height>571</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: nane;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_frequency_payments_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>288</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	color: black;
  	font-size:14px;
  	border:none;
	border-radius: 5px;
	background-color: rgb(252, 248, 232);
}

QComboBox::hover {
	background-color: rgb(212, 226, 212);
	border-radius: 5px;
}

QComboBox::drop-down {
      subcontrol-origin: padding;
      subcontrol-position: top right;
      width: 15px;
      border:none;
 }

QComboBox QAbstractItemView {
	background:rgba(255,255,255,1);
    border:1px solid rgba(228,228,228,1);
    border-radius:0px 0px 5px 5px;
	 font-size:14px;
    outline: 0px;
}

QComboBox QAbstractItemView::item{
	height:36px;
	color:red;
	padding-left:9px;
	background-color:red;
}</string>
   </property>
   <item>
    <property name="text">
     <string>Разовое</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>376</y>
     <width>125</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(212, 226, 212);
    border-radius: 20px;
    border-color: beige;
}
QPushButton:pressed {
    background-color: rgb(182, 205, 182);
    border-style: inset;
}</string>
   </property>
   <property name="text">
    <string>Сбросить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
